privacyidea/static/components/token/views/token.list.html

```diff
@@ -11,6 +11,15 @@
 
 <span translate>total tokens:</span> {{ tokendata.count }}
 
+Browserlanguge {{ browserLanguage }}
```

Comment:
Is there a reason why this is in the productive html?

Reply:
Yeah. I did mean to do this.

It is there to be found by you! ;-)

Reply:
done in 262444ca